mbl/settings.py

```diff
@@ -167,3 +167,4 @@
 SOCIAL_AUTH_GITHUB_KEY = os.environ.get('SOCIAL_AUTH_GITHUB_KEY', None)
 SOCIAL_AUTH_GITHUB_SECRET = os.environ.get('SOCIAL_AUTH_GITHUB_SECRET', None)
 SOCIAL_AUTH_LOGIN_REDIRECT_URL = BASE_URL
+URI_NAMESPACE = BASE_URL
```

Comment:
this should be settable through an environment variable and can default to BASE_URL.

Reply:
Also, the change should also be made in production settings

Reply:
Made the following changes:
1. URI_NAMESPACE can be set through an environment variable , and BASE_URL is the default value
2. Added URI_NAMESPACE to production_settings.py file